constraints/miden-vm/range-checker.air

```diff
@@ -0,0 +1,31 @@
+mod RangeCheckerAir
+
+fn binary_not(e: scalar) -> scalar:
+    return 1 - e
+
+fn change(e: scalar) -> scalar:
+    return e' - e
```

Comment:
nit: In the Memory chiplet constraints, we used `delta`. I'm not sure that it's necessarily the best function name, but I'd prefer to try to be consistent in our naming across constraint modules